.env.example

```diff
@@ -0,0 +1,3 @@
+PRIVATE_KEY="<YOUR_PRIVATE_KEY>"
```

Comment:
What private key is this? Is this GOOGLE or GITHUB or ...? You should prefix the env names this kind of stuff.